src/Sarif.Multitool/Program.cs

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using CommandLine;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.WorkItems;
```

Comment:
>WorkItems [](start = 16, length = 9)

Why have these been added here? I don't see relevant changes that would require it. In any case, if truly unnecessary, we can let the automated code clean-up take care of it.

Reply:
When I removed the telemetry flush from this file, I forgot to remove the usings. I've removed them now.

---
In reply to: [396822720](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396822720) [](ancestors = 396822720)